docs/mp/testing.adoc

```diff
@@ -36,25 +47,77 @@ include::{rootdir}/includes/dependencies.adoc[]
 </dependency>
 ----
 
-== Usage - default
+== Usage
 A test can be annotated with `io.helidon.microprofile.tests.junit5.HelidonTest` annotation to mark it as a
 CDI test. This annotation will start the CDI container before any test method is invoked, and stop it after
 the last method is invoked. This annotation also enables injection into the test class itself.
 
+=== Usage - per method CDI container
+A test can be annotated as follows:
+
+`@HelidonTest(resetPerTest = true)`
+
+This will change the behavior as follows:
+
+- A new CDI container is created for each test method invocation
+- annotations to add config, beans and extension can be added for each method in addition to the class
+- you cannot inject fields or constructor parameters of the test class itself (as a single instance is shared by more containers)
+- you can add `SeContainer` as a method parameter of any test method and you will get the current container
+
+=== Usage - configuration
+In addition to the `@AddConfig` annotation, you can also use
+`@Configuration` to configure additional classpath properties config sources using `configSources`, and to
```

Comment:
In addition to the `@AddConfig` annotation, you can also use
`@Configuration` to configure additional classpath properties and `configSources` to mark that a custom configuration is desired.

Reply:
I'm not sure I understand this:
 to configure additional classpath properties config sources using `configSources`

Reply:
fixed:)